features/step_definitions/post_steps.rb

```diff
@@ -0,0 +1,47 @@
+Given(/^I create a post that says "(.*?)"$/) do |content|
+  visit "/posts/new"
+  fill_in "Content", with: Faker::Company.buzzword
```

Comment:
It looks like you're not actually using the `content` variable that's being passed in.